yarn.lock

```diff
@@ -1668,6 +1668,18 @@
     "@uifabric/set-version" "^7.0.23"
     tslib "^1.10.0"
 
+"@fluentui/react-conformance@^0.10.1":
```

Comment:
interesting behaviour why was this added to yarn.lock as it's a monorepo package

Reply:
Version mismatch, we have `0.11.0` while [packages/react-conformance-griffel/package.json](https://github.com/microsoft/fluentui/pull/21924/files/2f63a341f3f73e26e8612763cfac91a30a5c29f7#diff-74639d9196f568828a4067a1db7b52e57adc8490c38abd4a88540b4c497fe68e) has `0.10.1`. This happen because of a recent release.

https://github.com/microsoft/fluentui/blob/f2ecf6614a11aaf38d2d66bb0f61143487b74e6d/packages/react-conformance/package.json#L3

Until we will use [`workspace:` instead of exact versions](https://yarnpkg.com/features/workspaces#publishing-workspaces) it will happen time to time.

Reply:
damn... right I'm not fan of this DX either. but that feature is not supported by midgard-yarn afaik

Reply:
Correct, it was added in Yarn v2, https://github.com/yarnpkg/berry/blob/master/CHANGELOG.md#200

I also don't think that `beachball` supports it...